### PR TITLE
chore: add `v.mod` manifest file

### PR DESCRIPTION
**Pull request by ttytm**

### CODE REVIEW

**Review by elliotchance — APPROVED** [review]

File: v.mod [hunk 1]
```diff
@@ -0,0 +1,7 @@
+Module{
+	name: 'vsql'
+	description: 'Single-file or PostgreSQL-server compatible transactional SQL database written in pure V.'
+	version: '0.29.2'
```
Comment (elliotchance) [inline v.mod:4]: This is going to be really painful to update with every version, can it be left out?

**Review by elliotchance — CHANGES_REQUESTED** [review]

File: v.mod (hunk 1 above)
Comment (ttytm) [reply v.mod:4]: Thanks for the review. Having a v.mod file would help with better future compatibility when V progresses. Ofc its free choice to leave it out. Atm, I don't see a downside here of not having it.

Comment (elliotchance) [reply v.mod:4]: I'm fine with the file, but the the versions are tagged for every (functionality change) diff so updating this every time will be super painful. Can the version be left out?

Comment (ttytm) [reply v.mod:4]: What I forgot in my comment above is that having the file will allow users to vsql install via Vs package manger at a specific version from the git url. 

E.g., currently it is:
```
❯ v install https://github.com/elliotchance/vsql@v0.29.2
Scanning `https://github.com/elliotchance/vsql@v0.29.2`...
error: failed to find `v.mod` for `https://github.com/elliotchance/vsql@v0.29.2`.
```

Afaik it should work without the version in the `v.mod` file based on the git tag. Else automating the version change on a tag creation should be possible as well.

Comment (elliotchance) [reply v.mod:4]: Take out to the version for now and let's see if it works after landing.

**Review by elliotchance — APPROVED** [review]

**Comment by elliotchance** [comment]
@ttytm I won't tag a new version for this (since it's not a code change) but let me know if this works.

**Pull request merged**